modules.py

```diff
@@ -52,10 +60,27 @@ def dist(self, file_path='distortions.par'):
         self._dist = []
         for i in range(len(data)):
             self._dist.append(float(data[i].partition('#')[0].strip()))
+        logging.debug("distortions (@ 0.547um): %s" % self._dist)
+        logging.debug("distortions (@ %sum): %s" % (self.wavelength, [self._dist[i]*.547/self.wavelength for i in
+                                                    range(len(self._dist))]))
         return self._dist
 
     def save(self, name=None):
-        hdu = pyfits.PrimaryHDU(self.a)
+        """
+        Will save the data array "a" in a fits file with header information as follow:
+        :param name: name of the .fits file to be written. If None, will call it self.name
+        :return: None
+        """
+
+        if self.b is None:
+            hdu = pyfits.PrimaryHDU(self.a)
+        else:
+            hdu = pyfits.PrimaryHDU()
+            list_arrays = list()
+            list_arrays.append(self.a)
```

Comment:
I had expected that the data cube would be in one extension and the image of the polychromatic PSF would be in another?  Then the user can choose to either just use the 'POLYPSF' extension or the 'MONOPSF' one?